internal/database/permission_sync_jobs.go

```diff
@@ -544,10 +544,20 @@ func (s *permissionSyncJobStore) List(ctx context.Context, opts ListPermissionSy
 const countPermissionSyncJobsQuery = `
 SELECT COUNT(*)
 FROM permission_sync_jobs
+%s -- whereClause
 `
 
-func (s *permissionSyncJobStore) Count(ctx context.Context) (int, error) {
-	q := sqlf.Sprintf(countPermissionSyncJobsQuery)
+func (s *permissionSyncJobStore) Count(ctx context.Context, opts ListPermissionSyncJobOpts) (int, error) {
+	conds := opts.sqlConds()
+
+	var whereClause *sqlf.Query
+	if len(conds) != 0 {
```

Comment:
Nit: I guess `len` should not be negative and this looks better to my eyes.
```suggestion
	if len(conds) > 0 {
```